### DISCUSSION TITLE
tab组件左右滑动不触发onchange时间，是出于什么考虑？

### DISCUSSION THREAD

**Discussion by L1-corona** [hellof2e/quark-design#311] [Announcements]
tab组件左右滑动不触发onchange时间，是出于什么考虑？

**Comment by sanqi-med** [comment]
> tab组件左右滑动不触发onchange时间，是出于什么考虑？

提个 issue，我们修复下。或者感兴趣的话也可以提个PR修复。

**Reply by L1-corona** [reply]
我是菜鸡不太会，就不提交了

**Reply by L1-corona** [reply]
github很多功能用的不是很熟，提issue这种找不到入口。

**Reply by sanqi-med** [reply]
已修复，请使用最新版
